src/flows/shared/steps/OrganizeCollection/Editor/StagingArea.tsx

```diff
@@ -121,14 +121,9 @@ function StagingArea({ nftsRef, stagedItems }: Props) {
           <StyledStagedNftContainer width={DND_WIDTHS[columns]}>
             {stagedItems.map((stagedItem) => {
               const size = IMAGE_SIZES[columns];
-              if (isEditModeNft(stagedItem)) {
-                return (
-                  <SortableStagedNft
-                    key={stagedItem.id}
-                    nftRef={nftFragmentsKeyedByID[stagedItem.id]}
-                    size={size}
-                  />
-                );
+              const stagedItemRef = nftFragmentsKeyedByID[stagedItem.id];
+              if (isEditModeNft(stagedItem) && stagedItemRef) {
+                return <SortableStagedNft key={stagedItem.id} nftRef={stagedItemRef} size={size} />;
```

Comment:
this new logic is similar to the one used on L105 of this file

![image](https://user-images.githubusercontent.com/12162433/166866276-e9f20162-8919-4804-b62f-019d3c5d8a7a.png)